suncityla/app/layout.tsx

```diff
@@ -1,22 +1,10 @@
-import type { Metadata } from "next";
-import localFont from "next/font/local";
+import Navbar from "./components/Navbar";
 import "./globals.css";
-import Link from "next/link";
-
-const geistSans = localFont({
```

Comment:
I think we should keep these fonts for now. Next sets up fonts in a specific way so we should keep this and edit it later.

Reply:
Okay, I'll keep them, should we add different fonts?

Reply:
we can do that know or later, up to you.